pkg/v4/resource/chart/update.go

```diff
@@ -56,9 +56,13 @@ func (r *Resource) ApplyUpdateChange(ctx context.Context, obj, updateChange inte
 	}
 
 	if !reflect.DeepEqual(updateState, ResourceState{}) {
-		r.logger.LogCtx(ctx, "level", "debug", "message", "updating chart-operator chart")
+		r.logger.LogCtx(ctx, "level", "debug", "message",
+			fmt.Sprintf("updating chart-operator chart, release name %q, release version %q from channel %q",
+				updateState.ReleaseName,
+				updateState.ReleaseVersion,
+				chartOperatorChannel))
```

Comment:
I would rather write everything for logs in one line even if it is longer. IMO this would still be better than such an usual indentation. 

<img width="217" alt="mha_hart_mah_sole" src="https://user-images.githubusercontent.com/552769/42281101-dabd801c-7fa2-11e8-8100-4055ce8efce5.png">

Reply:
Sure, done.